src/settings/target/mod.rs

```diff
@@ -120,6 +120,113 @@ impl Manifest {
         Ok(manifest)
     }
 
+    pub fn generate(
```

Comment:
the change here is a refactor to use `Manifest::default()` and move this `pub fn` declaration to the top.